examples/lambda/function.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   forProvider:
     packageType: Image
     code:
-      imageURI: 609897127049.dkr.ecr.us-east-1.amazonaws.com/example:latest
+      imageURI: 360427250399.dkr.ecr.us-east-1.amazonaws.com/example:nodejs12.x
```

Comment:
Is this new AWS Account sensitive ?

Reply:
No, this was just for testing. I reverted the changes. Thanks for pointing it out.